server/lib/model/action.dart

```diff
@@ -45,19 +42,101 @@ class ChangeLineItemStatusAction implements Action {
       ..type = proto.Action_Type.CHANGE_LINE_ITEM_STATUS
       ..changeLineItemStatusParams = (proto.ChangeLineItemStatusParams()
         ..status =
-            proto.ChangeLineItemStatusParams_Status.valueOf(_statusValue));
+            proto.ChangeLineItemStatusParams_Status.valueOf(statusValue));
   }
+}
 
-  /// Changes the status of the line item using the DV360 client.
-  /// TODO(@thu5): Record the return status and implement run history.
-  @override
-  Future<void> run(DisplayVideo360Client client, Target target) async {
-    final lineItemTarget = target as LineItemTarget;
+/// A class that represents changing the line item bidding strategy.
+class ChangeLineItemBiddingStrategyAction implements Action {
+  /// The integer value of the bidding strategy.
+  int biddingStrategy;
```

Comment:
wondering can we use enum here, so later the switch case can use enum, which has better readability.

Reply:
Thank you! That makes it a LOT more readable.